### PR TITLE
Api status feature

### PR DESCRIPTION
**Pull request by kjowong**

### CODE REVIEW

File: api/v1/views/__init__.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+from flask import Blueprint
+app_views = Blueprint("app_views", __name__, url_prefix="/api/v1")
+from api.v1.views.index import *
```
Comment (KRLoyd) [inline api/v1/views/__init__.py:3]: We should move this to line 1, so the imports will be alphabetical

File: api/v1/views/index.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/python3
+"""
+    index.py file in v1/views
+"""
+from flask import Flask, jsonify
+from api.v1.views import app_views
```
Comment (KRLoyd) [inline api/v1/views/index.py]: move this line to line 5 so our imports are alphabetical

**Review by KRLoyd — COMMENTED** [review]
Code looks good, would just move those imports per comments

File: api/v1/views/__init__.py (hunk 1 above)
Comment (kjowong) [reply api/v1/views/__init__.py:3]: The order of import is important as we need to create the Blueprint and import that package. We need to leave it as is

File: api/v1/views/index.py (hunk 2 above)
Comment (kjowong) [reply api/v1/views/index.py]: Will update in latest pull

**Pull request closed without merging**